packages/buidler-core/src/internal/core/errors.ts

```diff
@@ -184,166 +293,173 @@ You probably imported @nomiclabs/buidler instead of @nomiclabs/buidler/config`
     INVALID_HD_PATH: {
       number: 107,
       message:
-        'HD path "%s" is invalid. Read BIP32 to know about the valid forms.'
+        "HD path %path% is invalid. Read about BIP32 to know about the valid forms."
     },
     INVALID_RPC_QUANTITY_VALUE: {
       number: 108,
       message:
-        "Received invalid value %s from/to the node's JSON-RPC, but a Quantity was expected."
+        "Received invalid value %value% from/to the node's JSON-RPC, but a Quantity was expected."
```

Comment:
Perhaps wrap the value in backticks to make it clear we got that exact value from the rpc:
```
"Received invalid value `%value%` from/to the node's JSON-RPC, but a Quantity was expected."
```